addons/crm/static/src/js/systray_activity_menu.js

```diff
@@ -0,0 +1,17 @@
+odoo.define('crm.systray.ActivityMenu', function (require) {
+"use strict";
+
+var ActivityMenu = require('mail.systray.ActivityMenu');
+
+ActivityMenu.include({
+    /**
+     * @override
+     */
+    _getViewsList(model) {
+        if (model === "crm.lead") {
+            return this._super.apply(this, arguments).concat([[false, 'calendar'], [false, 'pivot'], [false, 'graph'], [false, 'activity']]);
```

Comment:
I wonder if we need too call `_super` in this case, I feel like if the goal of the override is to make specific cases, they should be explicitly listed instead of relying on parent, but this is arguable.

If we add or remove a view in the parent, will we want to add/remove it here too by default or not?

Just raising the question, I don't think there is a good or bad answer.

If we keep it, I would just call `_super` once at the top, store result in a variable, do the condition altering the result, and then return it only once at the end.

Reply:
I think your idea to use explicitly listed items is indeed a good idea. 
Since I had to show the views in a specific order my solution became a little bit overkill as you can see : 

![overkillsolution](https://user-images.githubusercontent.com/49784121/80603367-9a4ca900-8a30-11ea-9dfc-d721a26a76e6.png)

![overkill_com](https://user-images.githubusercontent.com/49784121/80603416-af293c80-8a30-11ea-940f-20adca137604.png)